e2etests/web/regular_integration_tests/README.md

```diff
@@ -5,12 +5,16 @@ configuration (e.g. PWA vs non-PWA packaging), please create another
 directory under e2etests/web. Otherwise tests such as text_editing, history,
 scrolling, pointer events... should all go under this package.
 
+Tests can be run on both 'release' and 'profile' modes. However 'release' mode
+will swallow the stack trace and only fails the test. Use 'profile' mode for
+trouble-shooting purposes.
+
 # To run the application under test for traouble shooting purposes.
 flutter run -d web-server lib/text_editing_main.dart --local-engine=host_debug_unopt
 
 # To run the Text Editing test and use the developer tools in the browser.
-flutter run --target=test_driver/text_editing_e2e.dart -d web-server --web-port=8080 --release --local-engine=host_debug_unopt
+flutter run --target=test_driver/text_editing_e2e.dart -d web-server --web-port=8080 --profile --local-engine=host_debug_unopt
 
 # To test the Text Editing test with driver:
-flutter drive -v --target=test_driver/text_editing_e2e.dart -d web-server --release --browser-name=chrome --local-engine=host_debug_unopt
+flutter drive -v --target=test_driver/text_editing_e2e.dart -d web-server --profile --browser-name=chrome --local-engine=host_debug_unopt
```

Comment:
why did you change to profile ?  I set it profile for treeshaking test to be able to parse main.dart.js

Reply:
The release more shortens the error stack trace: https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/foundation/assertions.dart#L501

We can still put test in both release and profile modes.

I just wanted to say, if you have an error, profile mode shows the full stack trace. I'll change the wording :)